src/__tests__/proposalVersions.int.test.ts

```diff
@@ -34,6 +35,67 @@ const createTestBaseFields = async () => {
 };
 
 describe('/proposalVersions', () => {
+	describe('GET /:proposalVersionId', () => {
+		it('requires authentication', async () => {
+			await request(app).get('/proposalVersions/1').expect(401);
+		});
+
+		it('returns exactly one proposal version selected by id', async () => {
+			const systemSource = await loadSystemSource();
+			await createOpportunity({ title: '🔥' });
+			const testUser = await loadTestUser();
+			await createProposal({
+				externalId: 'proposal-1',
+				opportunityId: 1,
+				createdBy: testUser.keycloakUserId,
+			});
+			await createApplicationForm({
+				opportunityId: 1,
+			});
+			const newProposalVersion = await createProposalVersion({
```

Comment:
While this follows some of the existing conventions, and therefore I don't think my objection is reason to hold up this PR, I prefer to get the database-generated IDs explicitly rather than making the assumption that ID 1 was inserted. Something for future PRs, perhaps.

Reply:
The same can be said for the other items -- it's all being loaded right above so there's no cost to just `const  opportunity = createOpportunity...` and then accessing `opportunity.id` in the appropriate places.

Also the prefix `new` here isn't contextually relevant so I'd just call it `proposalVersion` or `testProposalVersion`.